experiment/test_config.yaml

```diff
@@ -435,51 +435,51 @@ testSuites:
   alphafeatures:
     args:
     - --timeout=180m
+    - --ginkgo.focus=\[Feature:DynamicKubeletConfig\]
     envs:
-    - GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:DynamicKubeletConfig\]
   autoscaling:
     args:
     - --timeout=300m
+    - --ginkgo.focus=\[Feature:ClusterSizeAutoscalingScaleUp\]|\[Feature:ClusterSizeAutoscalingScaleDown\] --ginkgo.skip=\[Flaky\]
```

Comment:
wonder if one line is ok, or it need to be two flags --ginkgo.focus and --ginkgo.skip separately